libuavcan_drivers/linux/CMakeLists.txt

```diff
@@ -43,7 +43,7 @@ endif ()
 # Applications - tests, tools.
 #
 include_directories(include)
-set(CMAKE_CXX_FLAGS "-Wall -Wextra -pedantic -std=c++11")  # GCC or Clang
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall -Wextra -pedantic -std=c++11")  # GCC or Clang
```

Comment:
Can you maybe look into add_compile_options? That would scope the flags to libuavcan and not pollute other directories